lib/src/utils/utils.dart

```diff
@@ -9,10 +10,16 @@ class Utils {
     FocusScope.of(context).requestFocus(FocusNode());
   }
 
-  static void showAlertError({Object? error, required BuildContext context}) {
+  static DateTime? fromTimestamp(Timestamp? timestamp) => timestamp?.toDate();
+  static dynamic toTimestamp(DateTime? time) =>
```

Comment:
Please do not use dynamic type. I think what you need is sealed class and pattern matching. (similar to the ```Union Type``` of Typescript)

Reply:
@mafreud dynamic because it have two type return:

Timestamp if param not null
FieldValue.serverTimestamp if param null

Reply:
Ok then, you can use sealed class and pattern matching. They are much safer.

Reply:
@mafreud i removed this func, add new class timestamp converter